packages/server/modules/webhooks/domain/operations.ts

```diff
@@ -3,12 +3,26 @@ import { Webhook } from '@/modules/webhooks/domain/types'
 export type CreateWebhook = (
   webhook: Pick<
     Webhook,
-    'id' | 'streamId' | 'url' | 'description' | 'secret' | 'enabled'
-  > & { triggers: string[] }
+    'id' | 'streamId' | 'url' | 'description' | 'secret' | 'enabled' | 'triggers'
+  >
 ) => Promise<string>
 
 export type CountWebhooksByStreamId = ({
   streamId
 }: Pick<Webhook, 'streamId'>) => Promise<number>
 
 export type GetWebhookById = ({ id }: Pick<Webhook, 'id'>) => Promise<Webhook | null>
+
+export type UpdateWebhook = ({
+  webhookId,
+  webhookInput
+}: {
+  webhookId: string
+  webhookInput: Pick<Webhook, 'updatedAt'> &
```

Comment:
updatedAt, is prob not needed here, it should be the service's responsibility to craft that date properly imho

Reply:
yes, that's why it is required.
I mean the type says that the `updatedAt` field is required.
Does it make sense?